README.md

```diff
@@ -1 +1,16 @@
-# eslint-config-hss
\ No newline at end of file
+# eslint-config-hss
+
+Hearsay Social's base JavaScript style.
+
+## Ussage
+
+Install this package, and eslint:
+
+    npm install --save-dev eslint eslint-config-hss
```

Comment:
Indenting by four spaces triggers the code formatting as well:

https://daringfireball.net/projects/markdown/syntax#precode